packages/wallets/src/components/DerivAppsSection/DerivAppsSuccessFooter.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import useDevice from '../../hooks/useDevice';
+import { WalletButton } from '../Base';
+import { useModal } from '../ModalProvider';
+import './DerivAppsSuccessFooter.scss';
+
+const DerivAppsSuccessFooter = () => {
+    const { hide } = useModal();
+    const { isDesktop } = useDevice();
+    const history = useHistory();
+
+    return (
+        <div className='wallets-deriv-apps-success-modal__footer'>
```

Comment:
as per BEM methodology - className should be in line with component name, 
ideally DerivAppsSuccessFooter, but given our linting, wallets-deriv-apps-success-footer,

Reply:
Updated.